package.json

```diff
@@ -42,7 +42,7 @@
     "codeclimate-test-reporter": "^0.5.0",
     "ember-cli": "^3.10.1",
     "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-addon-docs": "^0.6.13",
+    "ember-cli-addon-docs": "^0.6.0",
```

Comment:
if this is a rollback two things:

* the `^` infront should allow the version to still drift up... what is the goal of this specific change, what version do we need?
* if an upstream issue exists, is there a corresponding issue?

Reply:
1. You were right. it doesn't make any changes. I remade changes to use ^0.6.* so that it's more general. 
2. There isn't a corresponding issue that I'm aware of.

I think this pr is to update yarn.lock so that ember-exam uses latest version of ember-cli-addon-docs along with latest version of ember-data. I will change description :)